decode_aprs.h

```diff
@@ -24,6 +24,8 @@ typedef struct decode_aprs_s {
 
 	int g_quiet;			/* Suppress error messages when decoding. */
 
+	int g_metric;			/* Print metric instead of US customary units */
```

Comment:
I would be more politically neutral:
```suggestion
	int g_metric;			/* Print metric instead of imperial units */
```